### PR TITLE
Improve dashboard ux

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
- [x] Redesign job list item
- [x] Move header help to bottom of the page
- [x] Rearrange 'add a job' button
- [x] Display only job url. Don't show source type

### CODE REVIEW

File: app/assets/stylesheets/_star-radios.scss [hunk 1]
```diff
@@ -23,14 +23,16 @@
...
+      font-size: 1.25em;
     }
 
     &.rating-options-group-enabled {
       /***** CSS Magic to Highlight Stars on Hover *****/
       > input:checked ~ label, /* show gold star when clicked */
       &:not(:checked) > label:hover, /* hover current star */
       &:not(:checked) > label:hover ~ label {
-        color: #FFD700;
+        color: $checked-star-color;
```
Comment (nathancashmore) [inline app/assets/stylesheets/_star-radios.scss]: Or just yellow ? as opposed to where its used, what it is.

File: app/assets/stylesheets/style.scss [hunk 2]
```diff
@@ -209,3 +175,20 @@ button.link-button {
     margin-top: $gutter-half;
   }
 }
+.job-item {
+  p {
+    margin-bottom: 0.4em;
```
Comment (nathancashmore) [inline app/assets/stylesheets/style.scss]: Is there a variable defined for this ? reuse if possible.

File: app/views/dashboard.mustache [hunk 3]
```diff
@@ -3,102 +3,80 @@
...
+        <div class="column-one-half">
+          <a href="{{basePath}}/{{accountId}}/jobs/new" class="button">{{#__}}dashboard.add-job-button{{/__}}</a>
+        </div>
 
-    <a href="{{basePath}}/{{accountId}}/jobs/new" class="button">{{#__}}dashboard.add-job-button{{/__}}</a>
+        {{#hasJobs}}
+        <div class="column-one-half tablet-right mobile-space-top">
+          <form action="{{basePath}}/{{accountId}}" method="GET">
+            <label for="sort">{{#__}}dashboard.sort.label{{/__}}</label>
+            <select class="form-control form-control-auto" name="sort" id="sort"
```
Comment (nathancashmore) [inline app/views/dashboard.mustache]: Do we need both name and id here ?

Comment (lukasz-kaniowski) [inline app/views/dashboard.mustache]: id is used for label `for` attribute

File: app/assets/stylesheets/style.scss (hunk 2 above)
Comment (lukasz-kaniowski) [inline app/assets/stylesheets/style.scss]: there is not. this variable `p` is scoped only for job-item as everything there is like a house of cards. you change this variable to `0.3em` and everything falls apart.

File: app/assets/stylesheets/_star-radios.scss (hunk 1 above)
Comment (lukasz-kaniowski) [inline app/assets/stylesheets/_star-radios.scss]: I would leave it as it is. It is used in 2 places and if we decide to update start color to something new you only have to change single line of code ( it is router in mine "code-line-nazi" nature)

**Pull request merged**